server/methods.js

```diff
@@ -0,0 +1,22 @@
+ Meteor.startup(function() {
+  if(Meteor.isServer) {
+    fuzzy = Npm.require("fuzzy");
+  }
+});
+
+ Meteor.methods({
+  findDreamCollegeId: function(userInputCollege) {
+    check(userInputCollege, String);
+    var colleges = CollegesInMemory;
+    var collegeNameArray = [];
+    var collegeNameHash = {}
+    _.each(colleges, function(college) {
+      collegeNameArray.push(college.name);
+      collegeNameHash[college.name] = college._id;
+    })
+    var results = fuzzy.filter(userInputCollege, collegeNameArray);
```

Comment:
This file's in `server/`, so shouldn't be an issue?

But FYI I did an implementation of this using mongo search in vanilla-node, for oli: https://github.com/AdmitHub/admithub-common/blob/vanilla-node/collections/profile.js#L645-L663

It's still a FIXME there because the mongo text search is also not available on client (unless we did some backflips with publications).